routes/overwrite.js

```diff
@@ -5,27 +5,56 @@ const router = express.Router()
 router.put('/', async (req, res, next) => {
 
   try {
-
+    
+    const overwriteBody = req.body
     // check for @id; any value is valid
-    if (!(req.body['@id'] ?? req.body.id)) {
+    if (!(overwriteBody['@id'] ?? overwriteBody.id)) {
       throw Error("No record id to overwrite! (https://store.rerum.io/API.html#overwrite)")
     }
 
-    const overwriteBody = JSON.stringify(req.body)
     const overwriteOptions = {
       method: 'PUT',
-      body: overwriteBody,
+      body: JSON.stringify(overwriteBody),
       headers: {
         'user-agent': 'TinyPen',
         'Origin': process.env.ORIGIN,
         'Authorization': `Bearer ${process.env.ACCESS_TOKEN}`,
         'Content-Type' : "application/json;charset=utf-8"
       }
     }
+
+    // Pass through If-Overwritten-Version header if present
+    const ifOverwrittenVersion = req.headers['if-overwritten-version']
+    if (ifOverwrittenVersion) {
+      overwriteOptions.headers['If-Overwritten-Version'] = ifOverwrittenVersion
+    }
+
+    // Check for __rerum.isOverwritten in body and use as If-Overwritten-Version header
+    const isOverwrittenValue = req.body?.__rerum?.isOverwritten
```

Comment:
I think just let the header be the trigger and remove this check.  It isn't doing much harm, but it means if I forget to scrub `__rerum` from the request body then I am triggering an optimistic locking overwrite.